src/app/screens/Onboard/TestConnection/index.tsx

```diff
@@ -29,7 +29,7 @@ export default function TestConnection() {
 
     try {
       const response = await api.getAccountInfo();
-      if (response.name && response.info.alias) {
+      if (response.name && response.balance?.balance) {
```

Comment:
if the balance is 0 this will return false

Reply:
should we also remove the name check?

Reply:
```suggestion
      if (response.name && response.balance?.balance !== 0) {
```